src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  value: [],
+};
+
+export const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkCategory: (state) => {
+      state.categories = 'Under construction';
+    },
+  },
+});
```

Comment:
- [ ] Kindly make sure in the `checkCategory`, you are changing the right property in the state object. You changed the `state.categories` while in your `initialState` object you only have the `value` property. Kindly fix this.

```suggestion
const initialState = {
  categories: [],
};

export const categoriesSlice = createSlice({
  name: 'categories',
  initialState,
  reducers: {
    checkCategory: (state) => {
      state.categories = 'Under construction';
    },
  },
});
```